### DISCUSSION TITLE
BlueSCSI compatibility

### DISCUSSION THREAD

**Discussion by Grieverheart** [ArrestedLightning/dietSCSI#2] [General]
Out of curiosity, the USB image passthrough is really interesting, so I was wondering if your firmware is compatible with BlueSCSIv1.

**Comment by ArrestedLightning** [comment]
Yes, that should work.  Keep in mind that some blue pill boards have the wrong resistor on the D+ line (10K instead of 1.5K), which may need to be fixed before USB will function correctly.  Also, some blue pill boards have fake/cloned STM32s on them, which can have problems with USB as well.  But it shouldn't hurt anything to give it a try.
